README.md

```diff
@@ -31,12 +31,19 @@ See the test.sh example...
 . ./encpass.sh
 password=$(get_password)
 # Call it specifying a directory
-#password=$(get_password ~/.ssh)
+#password=$(get_password -f ~/.ssh)
 echo $password
 ```
 
-## Limitations
```

Comment:
We do not want to get rid of POSIX compliance and for reasons outlined elsewhere we don't want to introduce options for SSH right now as the SSH dependency will be going away.